assets/src/scripts/monitoring-location/components/daily-value-hydrograph/graph-controls.js

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
+import {select} from 'd3-selection';
+import {Actions} from '../../store/daily-value-time-series';
+
+/*
+ * Draw the parameter code toggle for the daily values time series graph.
+ * @param {Object} elem - D3 selection
+ * @param {Object} store - Redux Store
+ */
+const drawDVTimeSeriesSelection = function(ul, store) {
+
+    const availTimeSeries = store.getState()
+        .dailyValueTimeSeriesData.availableDVTimeSeries;
+    const groupedTs = _.groupBy(availTimeSeries, (ts) => {
+        return ts.parameterCode;
+    });
+    const groupedKeys = Object.keys(groupedTs);
+    let checked = false;
+
+    if (groupedKeys.length > 1) {
+        let divSize = 30;
+        ul.append('li')
+            .text('Parameter Codes');
+
+        groupedKeys.forEach((key) => {
+
+            const paramCd = groupedTs[key][0].parameterCode;
+            const ts_id = groupedTs[key][0].id.split('-')[2];
+            const monitorLoc = `${groupedTs[key][0].id.split('-')[0]}-${groupedTs[key][0].id.split('-')[1]}`;
```

Comment:
You could pass in the monitoring location since it is static for the page. If you are going to split it out here it would be better to split the id once, assign to a constant and then use that to get the ts_id and monitorLoc instead of 4 times.